library/core/src/intrinsics/mod.rs

```diff
@@ -3982,6 +3982,52 @@ pub const fn minnumf64(x: f64, y: f64) -> f64;
 #[rustc_intrinsic]
 pub const fn minnumf128(x: f128, y: f128) -> f128;
 
+/// Returns the minimum (IEEE 754-2019 minimum) of two `f16` values.
```

Comment:
Given that we now have `minimum` and `minnum` (which also look extremely similar :/ ), please adjust the docs for the existing `minnum` intrinsics to clarify which kind of minimum they refer to.

Reply:
Sure, docs adjusted for the old intrinsics.